src/test/java/Homework16.java

```diff
@@ -0,0 +1,50 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import org.testng.xml.SuiteGenerator;
+import org.w3c.dom.html.HTMLInputElement;
+
+import java.time.Duration;
+
+
+public class Homework16 extends BaseTest {
+
+    @Test
+    public void registrationNavigationTest() {
+
+            ChromeOptions options = new ChromeOptions();
+            options.addArguments("--remote-allow-origins=*");
+
+            WebDriver driver = new ChromeDriver(options);
+            driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(10));
+
+            String url = "https://qa.koel.app";
+            driver.get(url);
+
+            WebElement registrationLink = driver.findElement(By.cssSelector("[href='registration']"));
+            registrationLink.click();
+            WebElement registrationText = driver.findElement(By.xpath("//div[@class='login-wrapper']/h2"));
+            Assert.assertTrue(registrationText.isDisplayed());
+            driver.quit();
+
+
+
+        //   WebElement emailInput = driver.findElement(By.cssSelector("[type='email']"));
+        //    WebElement passwordInput = driver.findElement(By.cssSelector("[type='password']"));
+        //    WebElement loginButton = driver.findElement(By.cssSelector("[type='submit']"));
+        //    emailInput.sendKeys("demo@class.com");
+        //    passwordInput.sendKeys("te$t$tudent");
+       //     loginButton.click();
+
+//            WebElement avatarIcon = driver.findElement(By.cssSelector("img[class='avatar']"));
+//            Assert.assertTrue(avatarIcon.isDisplayed());
+//            driver.quit();
```

Comment:
I recommend not using code comments: for, example, when you have a code that cannot be understood, then comments are not the solution. Try to rewrite the code or rename fields and other elements in a way that the reader will understand the action that you are doing there.